src/recordlinker/schemas/pii.py

```diff
@@ -209,7 +232,8 @@ def model_construct(
         obj.address = [Address.model_construct(**a) for a in values.get("address", [])]
         obj.name = [Name.model_construct(**n) for n in values.get("name", [])]
         obj.telecom = [Telecom.model_construct(**t) for t in values.get("telecom", [])]
-        obj.drivers_license = DriversLicense.model_construct(**values.get("drivers_license", {}))
+        obj.identifiers = [Identifier(**i) for i in values.get("identifiers", [])]
```

Comment:
```suggestion
        obj.identifiers = [Identifier.model_construct(**i) for i in values.get("identifiers", [])]
```

Reply:
I was using the model construct initially but it was not mapping the IdentifierType to an enum and was leaving it as a string.